python/tvm/relay/frontend/tflite.py

```diff
@@ -64,7 +64,12 @@ def __init__(self, model, subgraph, exp_tab):
             'MAX_POOL_2D': self.convert_max_pool2d,
             'CONCATENATION': self.convert_concatenation,
             'ADD': self.convert_add,
+            'SUB': self.convert_sub,
```

Comment:
how about `self._convert_elemwise(_op.subtract, op)` here?

Reply:
I prefer current way. Because when we have quantized support, we could not simply pass self._convert_elemwise. We use convert_sub could let us get quantization parameter from tflite tensor and decide what we should do. The second reason we could make it clearly. i.e. when we meet xxx op, we have convert_xxx function.